HackIllinois/UI/HILabel.swift

```diff
@@ -40,8 +40,8 @@ class HILabel: UILabel {
         case profileUsername
```

Comment:
Small nit: a lot of these files have long lists of cases that we can switch on. I definitely recommend updating the linter to alphabetize these lists so they are easier to read and update.